docs/ftldns/compile.md

```diff
@@ -27,16 +27,34 @@ sudo dnf install git wget ca-certificates gcc gmp-devel gmp-static m4 cmake libi
 Compile and install a recent version using:
 
 ```bash
-wget https://ftp.gnu.org/gnu/nettle/nettle-3.8.1.tar.gz
-tar -xzf nettle-3.8.1.tar.gz
-cd nettle-3.8.1
-./configure --libdir=/usr/local/lib
+wget https://ftp.gnu.org/gnu/nettle/nettle-3.9.tar.gz
+tar -xzf nettle-3.9.tar.gz
+cd nettle-3.9
+./configure --libdir=/usr/local/lib --enable-static --disable-shared --disable-openssl --disable-mini-gmp -disable-gcov --disable-documentation
 make -j $(nproc)
 sudo make install
 ```
 
 Since Ubuntu 20.04, you need to specify the library directory explicitly. Otherwise, the library will be installed in custom locations where it would not be found by `cmake`.
 
+## Compile `libmbedtls` from source
+
+*FTL*DNS uses another cryptographic library (`libmbedtls`) containing cryptographic primitives, X.509 certificate manipulation and the SSL/TLS and DTLS protocols used for serving the web interface and the API over HTTPS.
+
+Compile and install a recent version using:
+
+```bash
+wget https://github.com/Mbed-TLS/mbedtls/archive/refs/tags/v3.4.1.tar.gz -O mbedtls-3.4.1.tar.gz
```

Comment:
Update to 3.5.0

Reply:
We're not using 3.5.0. at the moment.

https://github.com/pi-hole/docker-base-images/blob/7d18911ef81739e05eade2c53b0a215097e71d1a/ftl-build/alpine/Dockerfile#L8

Reply:
Perhaps it was just me, but I wasn't able to compile on Debian 12 in Hyper-V using 3.4.1 until I moved to 3.5.0. Threw pseudo ops errors. Perhaps the 3.5.0 version was backwards compatible with an interim version that worked?

Reply:
We updated just recently, I will update the library version now